lib/rester/rspec.rb

```diff
@@ -47,17 +47,20 @@
         "Could not find path: #{path.inspect} verb: #{verb.inspect} context: #{context.inspect} in #{@rester_stub_filepath}"
     end
 
-    ex.example_group.let(:subject) {
+    ex.example_group.let(:request) {
```

Comment:
Calling this `request` is a little confusing to me.  Since this is really response from the service instead of the stub_response.

Maybe call this `service_response`?